Makefile

```diff
@@ -175,7 +185,7 @@ bin:
 	mkdir -p bin
 
 bin/regcopy: bin ## App to copy docker images to the Deckhouse registry
-	cd tools/regcopy; go build -o bin/regcopy
+	cd tools/regcopy; go get -a; go build -o bin/regcopy
```

Comment:
What does `go get -a` in this context?

Reply:
Forces to download missing modules. Without it, regcopy failed to build on my (fresh) machine